src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -254,6 +254,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_FeatureRecursivePatterns:
                 case MessageID.IDS_FeatureUsingDeclarations:
                 case MessageID.IDS_FeatureStaticLocalFunctions:
+                case MessageID.IDS_DefaultInterfaceImplementation:
```

Comment:
comment for `// semantic check` for consistency?

Reply:
> comment for // semantic check for consistency?

Sure. I'll add a comment in one of the next PRs

---
In reply to: [251111631](https://github.com/dotnet/roslyn/pull/32796#discussion_r251111631) [](ancestors = 251111631)